composer.json

```diff
@@ -570,4 +573,4 @@
             ]
         }
     }
-}
+}
```

Comment:
Looks like we're removing the newline. It's a super nit, but it seems like this shouldn't happen.

Reply:
Thanks! Addressed in this PR: https://github.com/googleapis/google-cloud-php/pull/5575